lektor/cli.py

```diff
@@ -291,8 +291,7 @@ def deploy_cmd(ctx, server, output_path, **credentials):
         event_iter = publish(env, server_info.target, output_path,
                              credentials=credentials, server_info=server_info)
     except PublishError as e:
-        raise click.UsageError('Server "%s" is not configured for a valid '
-                               'publishing method: %s' % (server, e))
+        raise click.UsageError('%s' % e)
```

Comment:
It looks like you're just trying to turn the PublishError into a string. This seems like a clearer way to do that:
```python
raise click.UsageError(str(e))
```